src/__tests__/utils.test.ts

```diff
@@ -72,11 +74,15 @@ describe('pick', () => {
 
 describe('formatDate', () => {
   it('should return `undefined` if an invalid date is given', () => {
-    expect(formatDate(null, 'DD/MM/YYYY')).toBe(undefined);
+    expect(formatDate(null, 'DD/MM/YYYY', null)).toBe(undefined);
+  });
+
+  it('should format correctly if a valid date is given', () => {
+    expect(formatDate(dateTest, 'DD/MM/YYYY', null)).toBe('21/06/2018');
   });
 
   it('should format correctly if a valid date is given', () => {
```

Comment:
Can you rename this test to something like `should format correctly if a valid date and a locale are given`.
Just so we don't have two tests with the same name.

Reply:
Done